liquibase-standard/src/main/java/liquibase/command/CommandScope.java

```diff
@@ -203,74 +206,96 @@ public CommandResults execute() throws CommandExecutionException {
         Scope.getCurrentScope().addMdcValue(MdcKey.OPERATION_START_TIME, Instant.ofEpochMilli(operationStartTime.getTime()).toString());
         // We don't want to reset the command name even when defining another CommandScope during execution
         // because we intend on keeping this value as the command entered to the console
+        String commandName = String.join(" ", commandDefinition.getName());
         if (!Scope.getCurrentScope().isMdcKeyPresent(MdcKey.LIQUIBASE_COMMAND_NAME)) {
-            Scope.getCurrentScope().addMdcValue(MdcKey.LIQUIBASE_COMMAND_NAME, String.join(" ", commandDefinition.getName()));
+            Scope.getCurrentScope().addMdcValue(MdcKey.LIQUIBASE_COMMAND_NAME, commandName);
         }
-        CommandResultsBuilder resultsBuilder = new CommandResultsBuilder(this, outputStream);
-        final List<CommandStep> pipeline = commandDefinition.getPipeline();
-        final List<CommandStep> executedCommands = new ArrayList<>();
-        Optional<Exception> thrownException = Optional.empty();
-        validate();
+        Event analyticsEvent = ExceptionUtil.doSilently(() -> {
+            return new Event(commandName);
+        });
```

Comment:
This event should only be created if telemetry is enabled. This will be handled in a future story.